packages/devtools_app/lib/src/screens/network/network_request_inspector.dart

```diff
@@ -78,6 +78,18 @@ class NetworkRequestInspector extends StatelessWidget {
               _buildTab(tabName: NetworkRequestInspector._cookiesTabTitle),
           ],
         ];
+        final tabViews = [
+          if (data != null) ...[
```

Comment:
this will throw an exception when data is null because the length of `tabs` will not match `tabViews`. If you put this back where it was, this code won't be reached when data is null. Another option is to make this a function

`List<Widget> generateTabViews() => [...];` so that way we don't pay the cost of creating all these objects when they aren't needed when `data` is null. Same would apply to the `tabs` list

Reply:
ooo or a better idea. Make `tabs` and `tabViews` late final so they are only generated when the variables are used